test/IRGen/debug_variable.sil

```diff
@@ -0,0 +1,25 @@
+// RUN: %target-swiftc_driver -g -emit-ir %s | %FileCheck %s
```

Comment:
All the other IRGenDebugInfo tests are in the test/DebugInfo directory. It would make sense to move this test there too.